src/mjolnir/osmpbfparser.cc

```diff
@@ -285,8 +285,4 @@ void Parser::parse(std::ifstream& file, const Interest interest, Callback& callb
   delete [] unpack_buffer;
 }
 
-void Parser::free() {
-  google::protobuf::ShutdownProtobufLibrary();
```

Comment:
so after you call this you basically can never use protobuf objects again in your program. because i need to build tiles and then also use protobuf (for communications within the actor) i cant have this. reading the docs it seems ok to not call this at the cost of a small amount of memory